python/res/enkf/enkf_simulation_runner.py

```diff
@@ -77,11 +77,11 @@ def runEnsembleExperiment(self, job_queue, run_context):
         """ @rtype: int """
         return self.runSimpleStep(job_queue, run_context)
 
-
-    def runWorkflows(self , runtime):
+    @staticmethod
+    def runWorkflows( runtime, ert):
         """:type res.enkf.enum.HookRuntimeEnum"""
-        hook_manager = self._enkf_main().getHookManager()
-        hook_manager.runWorkflows(runtime  , self._enkf_main())
+        hook_manager = ert.getHookManager()
+        hook_manager.runWorkflows(runtime  , ert)
```

Comment:
Just discussed this with @oyvindeide 
This should work, and we think we should push it further. Let's get rid of this module entirely. Almost all of the methods here can be made free functions (by passing in the `ert` instance) , and we can move them to `enkf_main.py`
The only method that cannot be made free is `createRunPath`, but that method is already available in the EnkfMain class (with a slightly different uppercase/lowercase combination)